lib/rules/require-curly-braces.js

```diff
@@ -72,7 +72,7 @@ module.exports.prototype = {
         if (typeIndex['case'] || typeIndex['default']) {
             file.iterateNodesByType('SwitchCase', function(node) {
                 // empty case statement
-                if (!node.consequent || node.consequent.length === 0) {
+                if (node.consequent.length === 0) {
```

Comment:
Right. My bad, I was thinking about `SwitchStatement`, not `SwitchCase`. This seems okay then.